src/frontend/src/components/PageLayout.tsx

```diff
@@ -23,20 +27,15 @@ const PageLayout: React.FC<PageLayoutProps> = ({
   tabs
 }) => {
   return (
-    <Fragment>
+    <Box>
+      <CssBaseline />
```

Comment:
what's this for?

Reply:
I saw it in the MUI documentation but i dont think i need it anymore